zx.mjs

```diff
@@ -31,12 +31,13 @@ await async function main() {
   if (typeof argv.prefix === 'string') {
     $.prefix = argv.prefix
   }
+  $.log = argv['log-stdout'] ? (...args) => console.log(...args) : (...args) => console.error(...args)
```

Comment:
Let's not introduce a new config function. Let's try to find out a better solution. Probably we can agree on what log of commands should always go to stderr.